### PR TITLE
#328-329 Mobile add/edit task/event form

### PR DESCRIPTION
**Pull request by ghost**
updates styles and views for create/edit tasks and generating slots
add configuration for rz-slider 
delete redundant styles
minor updates for header 
update datetimepicker with styles from develop (desktop branch). still need to configure it

### CODE REVIEW

File: mobile/modules/core/less/_core.datepicker.less [hunk 1]
```diff
@@ -15,18 +15,18 @@
       thead {
```
Comment (ghost) [inline mobile/modules/core/less/_core.datepicker.less:15]: note for file: only updates from develop branch

File: mobile/modules/core/less/_core.datepicker.less [hunk 2]
```diff
@@ -15,18 +15,18 @@
...
         tr:first-child th {
           border-radius: 0;
-          background: @color-mountain-meadow-green-s85;
+          background: @color-biscay-blue-s41;
```
Comment (lotrien) [inline mobile/modules/core/less/_core.datepicker.less:18]: Could you please use component variable

File: mobile/modules/core/less/_core.datepicker.less [hunk 3]
```diff
@@ -44,7 +44,7 @@
 
             &:not(.active) {
               font-weight: 900;
-              color: @color-mountain-meadow-green-s85;
+              color: @color-biscay-blue-s41;
```
Comment (lotrien) [inline mobile/modules/core/less/_core.datepicker.less:47]: Could you please use component variable

File: mobile/modules/core/less/_core.datepicker.less [hunk 4]
```diff
@@ -54,19 +54,246 @@
...
         }
       }
     }
   }
+}
+
+.bootstrap-datetimepicker-widget {
+  thead tr:first-child {
+    font-size: 1rem;
+    color: @color-white;
```
Comment (lotrien) [inline mobile/modules/core/less/_core.datepicker.less:69]: Could you please use component variable

File: mobile/modules/core/views/main/datetime-picker.client.view.html [hunk 5]
```diff
@@ -1,5 +1,6 @@
+<!-- todo refactor plugin control -->
 <label>{{label}}</label>
 <div class="input-group date datetime-picker" id="{{pickerId}}">
-	<span class="input-group-addon datetimepicker-button" data-ng-class="{'disabled': isReadonly}"><i class="fa fa-calendar"></i></span>
-    <input type="text" class="form-input" required data-ng-disabled="isReadonly"/>
-</div>
+	<span class="input-group-addon datetimepicker-button"><i class="fa fa-calendar"></i></span>
```
Comment (lotrien) [inline mobile/modules/core/views/main/datetime-picker.client.view.html:4]: Can we use icon not from font-awesome, but from material-icons?

File: mobile/modules/core/less/_core.datepicker.less (hunk 2 above)
Comment (ghost) [reply mobile/modules/core/less/_core.datepicker.less:18]: we do it in scope of #373 ticket

File: mobile/modules/core/less/_core.datepicker.less (hunk 3 above)
Comment (ghost) [reply mobile/modules/core/less/_core.datepicker.less:47]: we do it in scope of #373 ticket

File: mobile/modules/core/views/main/datetime-picker.client.view.html (hunk 5 above)
Comment (ghost) [reply mobile/modules/core/views/main/datetime-picker.client.view.html:4]: Yes, we do it in scope of #373 ticket

**Pull request merged**